### PR TITLE
{phys}[intel/2020a] Meep v1.4.3

### PR DESCRIPTION
**Pull request by ekieffer**
(created using `eb --new-pr`)

requires ~#11380~ (Harminv) and ~#11383~ (libctl)

### CODE REVIEW

**Comment by ekieffer** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-002 - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/13c5ac7b141d918ab5473b7905f8d467 for a full test report.

**Comment by Falkor** [comment]
Depends on PR #11380 now merged

**Comment by boegel** [comment]
close/re-open to re-trigger tests...

File: easybuild/easyconfigs/m/Meep/Meep-1.4.3-intel-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+name = 'Meep'
+version = '1.4.3'
+
+homepage = 'https://github.com/stevengj/meep'
+description = """Meep (or MEEP) is a free finite-difference time-domain (FDTD) simulation software package
+ developed at MIT to model electromagnetic systems."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'usempi': True, 'opt': True, 'optarch': True, 'unroll': True, 'pic': True}
```
Comment (boegel) [inline easybuild/easyconfigs/m/Meep/Meep-1.4.3-intel-2020a.eb:11]: `optarch` is enabled by default, no need to enable it

```suggestion
toolchainopts = {'usempi': True, 'opt': True, 'unroll': True, 'pic': True}

File: easybuild/easyconfigs/m/Meep/Meep-1.4.3-intel-2020a.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+configopts = "--with-pic --with-mpi --with-blas=mkl_em64t --with-lapack=mkl_em64t --without-gcc-arch "
+configopts += "--with-libctl=$EBROOTLIBCTL/share/libctl --enable-shared "
+
+# fix for guile-config being broken because shebang line contains full path to bin/guile
+configopts += 'GUILE_CONFIG="$EBROOTGUILE/bin/guile -e main -s $EBROOTGUILE/bin/guile-config"'
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['bin', 'lib']
+}
```
Comment (boegel) [inline easybuild/easyconfigs/m/Meep/Meep-1.4.3-intel-2020a.eb:39]: sanity check can/should be more specific:

```suggestion
    'files': ['bin/meep', 'include/meep.hpp', 'lib/libmeep.a', 'lib/libmeep.%s' % SHLIB_EXT, 'lib/pkgconfig/meep.pc'],
    'dirs': ['include/meep', 'share/meep'],
}

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2625.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/91e8e056b7e5d60bee2ec138b4475f92 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3516.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/2a0c951fd299d03f5e9c37d10255e7e3 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @ekieffer!

**Pull request merged**